kola/tests/kubeadm/kubeadm.go

```diff
@@ -174,7 +177,8 @@ func init() {
 					Run: func(c cluster.TestCluster) {
 						kubeadmBaseTest(c, testParams)
 					},
-					MinVersion: semver.Version{Major: major},
+					MinVersion: semver.Version{Major: majorMinVersion},
+					EndVersion: semver.Version{Major: majorEndVersion},
```

Comment:
@tormath1 thanks for the review, I think I added a decent check, not the best though, as it is in two places.